microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -287,4 +287,20 @@ Issued: 2024-05-09
 
 {product-title} release 4.15.12 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:2667[RHSA-2024:2667] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:2664[RHSA-2024:2664] advisory.
 
-For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deployment_microshift-embed-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
\ No newline at end of file
+For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deployment_microshift-embed-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
+
+[id="microshift-4-15-13-dp"]
+=== RHBA-2024:2775 - {microshift-short} 4.15.13 bug fix and security update advisory
+
+Issued: 2024-05-15
+
+{product-title} release 4.15.13 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:2775[RHBA-2024:2775] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:2773[RHSA-2024:2773] advisory.
+
+For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deployment_microshift-embed-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
+
+[discrete]
+[id="microshift-4-15-13-enhancements"]
+==== Enhancements
+
+GitOps with Argo CD documentation is now available::
+* With this release, documentation for using GitOps is now available. Using GitOps means you can update a single Git repository and automate the deployment of new applications or updates to existing ones. For more information, see xref:../microshift_install/microshift-install-rpm.adoc#microshift-installing-rpms-for-gitops_microshift-install-rpm[Installing the GitOps Argo CD manifests from an RPM package](link:https://issues.redhat.com/browse/OSDOCS-9729[OSDOCS-9729]) and xref:../microshift_running_apps/microshift-gitops.adoc#microshift-gitops[Automating application management with the GitOps controller](link:https://issues.redhat.com/browse/OSDOCS-9764[OSDOCS-9764])
```

Comment:
```suggestion
* With this release, documentation for using GitOps is now available. Using GitOps means you can update a single Git repository and automate the deployment of new applications or updates to existing ones. For more information, see xref:../microshift_install/microshift-install-rpm.adoc#microshift-installing-rpms-for-gitops_microshift-install-rpm[Installing the GitOps Argo CD manifests from an RPM package] and xref:../microshift_running_apps/microshift-gitops.adoc#microshift-gitops[Automating application management with the GitOps controller].
```

Reply:
This is not a bug fix, so we do not use the Jira issue here.

Reply:
okay